ansible/templates/common/cors_configuration.sh.j2

```diff
@@ -0,0 +1,17 @@
+{% if doc %}
+Configure CORS. An ``index`` to specify the ordering of middleware classes is used. It is important to add the ``CorsMiddleware`` as the first class and ``CorsPostCsrfMiddleware`` as the last. You can specify the origin allowed to access or specify all the origin, for example:
```

Comment:
Maybe "An index is used to specify the ordering of middleware classes."
And "You can specify allowed origins in a whitelist, or allow all, for example:"

Reply:
I think the last sentence still needs fixing "You can specify the origin allowed to access or specify all the origin, for example" -> "You can specify allowed origins in a whitelist, or allow all, for example"